contracts/v2/accounts/default/ECDSA.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.0;
+
+/**
+ * @dev Elliptic Curve Digital Signature Algorithm (ECDSA) operations.
+ *
+ * These functions can be used to verify that a message was signed by the holder
+ * of the private keys of a given address.
+ */
+library ECDSA {
```

Comment:
this is from OZ 4.0? 

should we use a never version? Quickly checked I don't think there is anything critical but between 4.0 and 4.8 there were some fixes that we should check at least a little better if we don't update 

Also not sure but do we have to copy in the license to the top of the file? I think not needed with MIT? https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.0/LICENSE

Reply:
OZ lib 4.0 has no other libs, so used that file. 

4.8 version has strings lib to also be included

Reply:
ok! shortly verified their commits, all good